Header header
uint8 target_id
float32 x
float32 y
float32 vx
float32 vy
float32 range
float32 velocity
float32 bearing
float32 snr
uint8 flags
